ArduPlane/events.cpp

```diff
@@ -131,6 +131,8 @@ void Plane::failsafe_long_on_event(enum failsafe_state fstype, ModeReason reason
         if(g.fs_action_long == FS_ACTION_LONG_PARACHUTE) {
 #if PARACHUTE == ENABLED
             parachute_release();
+            //stop motors to avoide parachute tangling
+            plane.arming.disarm(AP_Arming::Method::PARACHUTE_RELEASE);
```

Comment:
Should probably pass false aswell to skip disarm checks.

Reply:
your right, there innoucuous now, but we might change them in the future in which case this would break and it would probably take us ages to notice.

Reply:
changed...